src/build.d

```diff
@@ -895,7 +895,15 @@ auto detectModel()
     if (detectOS == "solaris")
         uname = ["isainfo", "-n"].execute.output;
     else if (detectOS == "windows")
-        uname = ["wmic", "OS", "get", "OSArchitecture"].execute.output;
+    {
+        version (Windows)
+        {
+            import core.sys.windows.winbase;
+            int is64;
+            if (IsWow64Process(GetCurrentProcess(), &is64))
+                return is64 ? "64" : "32";
+        }
```

Comment:
Perhaps `else static assert(false);` for prudence / clarity, though the current code will have the same effect due to not returning a value.

Reply:
You mean `else assert(false);`? Otherwise this would stop compiling on non-Windows systems, this is a runtime check and not compile-time check after all.